.github/workflows/ci-actions-incremental.yml

```diff
@@ -18,6 +20,10 @@ on:
       - '.github/*.conf'
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+    # TODO: revisit this in case incremental becomes the default
+    branches-ignore:
```

Comment:
This doesn't work as I was expecting it. In context of `pull_request`, this is actually checking the target branch, not the PR branch. Investigating...

Reply:
> this is actually checking the target branch

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags
> For a `pull_request` event, only branches and tags on the base are evaluated.

So the only way I see is to add a predicate to the `if` of each job in the workflow.

Reply:
`if`s added.